### PR TITLE
[Optimization-2349][core] Optimize dinky executor

### LINKED ISSUE #2349: [Optimization][core] Optimize dinky executor
### Search before asking

- [X] I had searched in the [issues](https://github.com/DataLinkDC/dinky/issues?q=is%3Aissue) and found no similar optimization requirement.

### Description

_No response_

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by aiwenmo**
## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

### CODE REVIEW

File: dinky-admin/src/main/java/org/dinky/controller/JobInstanceController.java [hunk 1]
```diff
@@ -106,6 +107,18 @@ public Result<JobInfoDetail> getJobInfoDetail(@RequestParam Integer id) {
...
+    @PostMapping("/getOneById")
+    @ApiOperation("Get job instance info by job instance id")
+    @ApiImplicitParam(
+            name = "id",
+            value = "Job instance id",
+            dataType = "Integer",
+            paramType = "query",
+            required = true)
+    public Result getOneById(@RequestBody ID id) {
+        return Result.succeed(jobInstanceService.getById(id.getId()), "获取成功");
```
Comment (Zzm0809) [inline dinky-admin/src/main/java/org/dinky/controller/JobInstanceController.java]: 这里不需要 加 msg 了

File: dinky-app/dinky-app-base/src/main/java/org/dinky/app/flinksql/Submitter.java [hunk 2]
```diff
@@ -170,13 +171,13 @@ public static void submit(Integer id, DBConfig dbConfig, String dinkyAddr) {
...
-        ExecutorSetting executorSetting = ExecutorSetting.build(taskConfig);
+        ExecutorConfig executorConfig = ExecutorConfig.buildFromMap(taskConfig);
 
         // 加载第三方jar
-        loadDep(taskConfig.get("type"), id, dinkyAddr, executorSetting);
+        loadDep(taskConfig.get("type"), id, dinkyAddr, executorConfig);
 
-        logger.info("作业配置如下： {}", executorSetting);
-        Executor executor = Executor.buildAppStreamExecutor(executorSetting);
+        logger.info("作业配置如下： {}", executorConfig);
```
Comment (Zzm0809) [inline dinky-app/dinky-app-base/src/main/java/org/dinky/app/flinksql/Submitter.java]: use en

File: dinky-app/dinky-app-base/src/main/java/org/dinky/app/flinksql/Submitter.java [hunk 3]
```diff
@@ -202,24 +203,24 @@ public static void submit(Integer id, DBConfig dbConfig, String dinkyAddr) {
...
         if (trans.size() > 0) {
-            if (executorSetting.isUseStatementSet()) {
+            if (executorConfig.isUseStatementSet()) {
                 List<String> inserts = new ArrayList<>();
                 for (StatementParam item : trans) {
                     if (item.getType().equals(SqlType.INSERT)) {
                         inserts.add(item.getValue());
                     }
                 }
                 logger.info("正在执行 FlinkSQL 语句集： " + String.join(FlinkSQLConstant.SEPARATOR, inserts));
```
Comment (Zzm0809) [inline dinky-app/dinky-app-base/src/main/java/org/dinky/app/flinksql/Submitter.java]: modify to en

File: dinky-app/dinky-app-base/src/main/java/org/dinky/app/flinksql/Submitter.java [hunk 4]
```diff
@@ -202,24 +203,24 @@ public static void submit(Integer id, DBConfig dbConfig, String dinkyAddr) {
...
-                executor.submitStatementSet(inserts);
+                executor.executeStatementSet(inserts);
                 logger.info("执行成功");
```
Comment (Zzm0809) [inline dinky-app/dinky-app-base/src/main/java/org/dinky/app/flinksql/Submitter.java]: modify to en

File: dinky-app/dinky-app-base/src/main/java/org/dinky/app/flinksql/Submitter.java [hunk 5]
```diff
@@ -202,24 +203,24 @@ public static void submit(Integer id, DBConfig dbConfig, String dinkyAddr) {
...
             } else {
                 for (StatementParam item : trans) {
                     logger.info("正在执行 FlinkSQL： " + item.getValue());
```
Comment (Zzm0809) [inline dinky-app/dinky-app-base/src/main/java/org/dinky/app/flinksql/Submitter.java]: modify to en

File: dinky-executor/src/main/java/org/dinky/executor/ExecutorConfig.java [hunk 6]
```diff
@@ -47,10 +47,10 @@
 @Setter
 @Getter
 @ApiModel(value = "ExecutorSetting", description = "Executor settings for a job")
```
Comment (Zzm0809) [inline dinky-executor/src/main/java/org/dinky/executor/ExecutorConfig.java]: change to `ExecutorConfig`

**Review by zackyoungh — APPROVED** [review]
LGTM

**Pull request merged**